demisto_sdk/commands/common/errors.py

```diff
@@ -2186,6 +2187,12 @@ def taskid_different_from_id(task_key, id_, taskid):
         return f"On task: {task_key},  the field 'taskid': {taskid} and the 'id' under the 'task' field: {id_}, " \
                f"must be with equal value. "
 
+    @staticmethod
+    @error_code_decorator
+    def incorrect_value_references(task_key, condition_name):
+        return f"On task: {task_key}, condition name: {condition_name}, make sure to change the reference to 'From previous tasks' " \
+            "from 'As value' when referencing a value from the context, or add '${' '}' to wrap the string"
```

Comment:
```suggestion
        return f"On task: {task_key}, condition name: {condition_name}, an input was passed as string, rather than as reference. Change the reference to 'From previous tasks' " \
            "from 'As value' when referencing a value from the context, or wrap it with  ${'<condition>'}"
```